site/content/blog/simplifying-package-authoring.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Simplifying Package Authoring"
+slug: simplifying-package-authoring
+date: 2022-08-17
+author: Rohit Aggarwal
+excerpt: "Simplifying Package Authoring with kctrl"
+image: /img/logo.svg
+tags: ['carvel', 'package', 'author', 'kctrl', 'create']
+---
+
+In Carvel, Kubernetes(K8s) manifest are distributed and consumed via the concept of Package. A package Author encapsulates, version, and distribute K8s manifest for package consumers to install on a Kubernetes cluster. A package author can choose to create a package by using third party manifest e.g. they can choose to create a package from cert-manager, dynatrace etc. or they can distribute their own project K8s manifest by creating a Package.
```

Comment:
nit: why is `package Author` capitalized like that?

Reply:
Adopted. Changed to `Package Author`